src/store/configureStore.dev.js

```diff
@@ -19,9 +19,9 @@ export default function configureStore(initialState) {
     thunkMiddleware,
   ];
 
-  const store = createStore(rootReducer, initialState, compose(
-    applyMiddleware(...middlewares),
-    window.devToolsExtension ? window.devToolsExtension() : f => f // add support for Redux dev tools
+  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose; // add support for Redux dev tools
```

Comment:
I think it's safe to assume `window` will be defined and even if you hit this code while running tests, `window` can be mocked in your tests.